extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -85,12 +85,24 @@ export class ViewerSubscriptionPlatform {
       'productId': this.currentProductId_,
       'origin': this.origin_,
     });
-    // TODO(chenshay): Viewer Matching: We don't know which viewer it actually
-    // is. Need to check the viewerUrl to know, or more specificlly iterate via
-    // configured platforms and check whether any of these support the viewer.
-    const encryptedDocumentKey = this.serviceAdapter_.getEncryptedDocumentKey(
+    // Defaulting to google.com for now.
+    // TODO(@elijahsoria): Remove this line and only rely on what is returned
+    // in the cryptokeys param.
+    let encryptedDocumentKey = this.serviceAdapter_.getEncryptedDocumentKey(
       'google.com'
     );
+    const cryptokeysNames = this.viewer_.getParam('cryptokeys');
```

Comment:
How about we just do this: `const cryptokeyNames = viewer.getParam(...) || 'google.com'` and then do the cycle and everything else.